### ISSUE TITLE
In JS highlighting, comments seem to be the same colour as variables

### ISSUE THREAD

**Issue by develohpanda** [seanghay/insomnia-plugin-github-light-theme#1]
This is such a good looking theme, has become my daily. Thank you for making it!

I noticed that in js highlighting, comments and variables have a deceptively similar color which threw me off. Is this something you have control over?

![image](https://user-images.githubusercontent.com/4312346/93407435-4ca0e200-f8e6-11ea-9433-94a09f292124.png)

**Comment by seanghay** [comment]
## Oops!

Sorry for the late reply! 

@develohpanda I honestly don't know which color to change, if you could help guide me which variable I should change to make it work, it would be great. I really don't understand much about colors using inside the code editor.

Thanks for using it, @develohpanda !

**Comment by develohpanda** [comment]
Looks like it's using `var(--hl)`, which I think is `highlight.default` (see [here](https://support.insomnia.rest/article/26-plugins#themes))

![image](https://user-images.githubusercontent.com/4312346/93952453-fbd53180-fd9c-11ea-944e-e7a1370bfcb2.png)

`cm-comment` uses `--hl`, but `cm-variable` uses `--color-warning`. The `expect` and `response` keywords in the screenshots above have the `cm-variable` class.

Perhaps you could explore changing these two variables for the editor and see what you like? Note the following is untested 😄 

```js
{
  styles: {
    editor: {
      highlight: {
        default: // something
      }
    }
  }
}
```
or
```js
{
  styles: {
    editor: {
      background: {
        warning: // something
      }
    }
  }
}
```
(If background doesn't work, could try set foreground)

**Comment by seanghay** [comment]
@develohpanda I found it now. 
![image](https://user-images.githubusercontent.com/15277233/94010860-b2063e80-fdd0-11ea-9058-a33c04e1bd3d.png)

**Comment by seanghay** [comment]
Fixed in `1.0.4` 🎉
